### ISSUE TITLE
Wrong calculation power consumption (+fix)

### ISSUE THREAD

**Issue by Janssuhhh** [enesbcs/Shelly_MQTT#34]
The calculation for the power consumption is wrong because the wrong value is used.

I believe the simple method to convert watt/min to kWh, implemented because of issue #26 , is actually a good one, besides the wrong value.
After some testing with a kWh-meter before a Plug S, I'm quite confident this is an easy fix.
On line 385:
             mval2 = round((mval***0.06**),4) # 10*Wh? or Watt-min??
Should be:
             mval2 = round((mval***0.017**),4) # 10*Wh? or Watt-min??

**Comment by enesbcs** [comment]
> The calculation for the power consumption is wrong because the wrong value is used.
> 
> I believe the simple method to convert watt/min to kWh, implemented because of issue #26 , is actually a good one, besides the wrong value.
> After some testing with a kWh-meter before a Plug S, I'm quite confident this is an easy fix.
> On line 385:
> mval2 = round((mval_**0.06**),4) # 10_Wh? or Watt-min??
> Should be:
> mval2 = round((mval_**0.017**),4) # 10_Wh? or Watt-min??

Interesting, is this working for you or can you tell me how can you calculate this 0.017 constant?

**Comment by enesbcs** [comment]
Modified in commit https://github.com/enesbcs/Shelly_MQTT/commit/b2f4f9bf369becd1a59e7cbe7a6492c7e0d138f7

**Comment by Janssuhhh** [comment]
A colleague told me the calculation was off and we started checking with different values.

As 1wattmin = 0,000017 kWh, we tried 0,017 instead of 0,6 and the values in the Shelly and in domoticz were showing the same.

So for us this is working. 

After checking with a kWh meter we see good values (apart from a 6%  higher value on the Shelly, but that is due to the Shellies not being 100% right, but that's not a problem of this script) 

That's why we believe this is a fix :-)
